### PR TITLE
Release armdatafactory v0.1.0

### PR DESCRIPTION
**Pull request by 804873052**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

**Comment by ArcturusZhang** [comment]
We have similar compile error in this package like in security

File: sdk/datafactory/armdatafactory/autorest.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+``` yaml
+require:
+- https://github.com/Azure/azure-rest-api-specs/blob/5d09c12c024fa7efbaca6a95b9741a46a886fe6f/specification/datafactory/resource-manager/readme.md
+- https://github.com/Azure/azure-rest-api-specs/blob/5d09c12c024fa7efbaca6a95b9741a46a886fe6f/specification/datafactory/resource-manager/readme.go.md
+module-version: 0.1.0
+directive:
+ - from: DataFlow.json
+   where: $.definitions.DataFlow
+   transform: >
+     $["required"] = ["type"]
```
Comment (jhendrixMSFT) [inline sdk/datafactory/armdatafactory/autorest.md:14]: This needs to be fixed in the swagger.

File: sdk/datafactory/armdatafactory/autorest.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ - from: ManagedPrivateEndpoint.json
+   where: $.definitions.ManagedPrivateEndpoint
+   transform: >
+     $["required"] = ["type"]
```
Comment (jhendrixMSFT) [inline sdk/datafactory/armdatafactory/autorest.md:18]: Schema `ManagedPrivateEndpoint` doesn't contain the field `type`.

File: sdk/datafactory/armdatafactory/autorest.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ - from: ManagedVirtualNetwork.json
+   where: $.definitions.ManagedVirtualNetwork
+   transform: >
+     $["required"] = ["type"]
```
Comment (jhendrixMSFT) [inline sdk/datafactory/armdatafactory/autorest.md:22]: Schema `ManagedVirtualNetwork` doesn't contain the field `type`.

**Pull request closed without merging**